pretty.c

```diff
@@ -837,4 +840,5 @@ void print_drm(struct json_object *obj)
 	json_object_object_foreach(obj, path, node_obj) {
 		print_node(path, node_obj);
 	}
+	print_modifier(216172782119018517ull);
```

Comment:
Looks like a line of testing code that wasn't supposed to make it in the commit.